tests/tests.rs

```diff
@@ -42,6 +45,12 @@ fn test_initialize() {
     assert!(Yara::new().is_ok());
 }
 
+#[test]
+fn test_configuration() {
+    assert_eq!(Ok(()), set_configuration(ConfigName::StackSize, 100));
+    assert_eq!(Ok(100), get_configuration(ConfigName::StackSize));
```

Comment:
It is safe to call `yr_[gs]et_configuration` without initializing libyara before?

Reply:
Yes, you can read source code here: https://github.com/VirusTotal/yara/blob/8206dc6f728fe50e21af92cb40e454b68ef6af05/libyara/libyara.c#L376

Reply:
It is not about the source code, but about the API, what Yara intend the user to do. However this is not clearly stated in the documentation.

Anyway, by looking at the code, I see two things:
- `yr_cfgs` is not initialized at the start of the program, so it might contain random value (or 0? I don't remember how C handle global variable initial values).
- `yr_cfgs` is initialized in [`yr_initialize`](https://github.com/VirusTotal/yara/blob/38ed07aaf28120e2f85582b4dc4ef28ac9344649/libyara/libyara.c#L293), so it is pointless to call `get` and `set_configuration` before calling `yr_initialize`.

Can you maybe hide this function and create a method in [`Yara`](https://github.com/Hugal31/yara-rust/blob/6d403653f6cf4551ac2530e11309b27c271c3445/src/lib.rs#L131)?

Reply:
Done